src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -19,20 +19,20 @@ export function getStaticProps(context) {
   };
 }
 
-This guide will help you migrate v5 Storage implementations to the new functional API's in v6.
+This guide will help you migrate v5 Storage implementations to the new functional APIs in v6.
 
 ## Deprecated Options
 
 <Callout>Many input and output properties have been deprecated due to low usage or added complexity. If you would like to see these options re-implemented in v6, feel free to open a [feature request](https://github.com/aws-amplify/amplify-js/issues/new?assignees=&labels=pending-triage&projects=&template=2.feature_request.yaml) in our github repository.</Callout>
 
-The input options below have been removed in v6 from Storage API's:
+The input options below have been removed in v6 from Storage APIs:
 
-- `provider`: _Custom providers are no longer supported in v6._
+- `provider`: _Custom providers are no longer supported in v6. All examples below use the input/output values expected from the Default AWSS3Provider._
```

Comment:
Nit(?): This seems to be saying that v6 samples are using AWSS3Provider but I think we just refer to it as the S3 Provider in v6